src/features/metadata-based-view/MetadataBasedItemList.js

```diff
@@ -314,14 +311,14 @@ class MetadataBasedItemList extends React.Component<Props, State> {
         return cellData;
     }
 
-    getGridHeaderData(columnIndex: number): string | Element<typeof FormattedMessage> {
-        const { metadataColumnsToShow } = this.props;
+    getGridHeaderData(columnIndex: number): string | Element<typeof FormattedMessage> | void {
+        const metadataFieldsToShow = this.getFieldsToShow();
 
+        if (columnIndex === 0) return undefined;
```

Comment:
Minor: can you add a comment for this addition? What was the `columnIndex === 0` case returning before?

Reply:
@sahiga Previously this got covered by `getMetadataColumnName()` and use of `lodash/get`